package.json

```diff
@@ -94,7 +94,8 @@
     "migrate-build-logs": "node ./scripts/migrate-build-logs.js",
     "send-email": "node ./scripts/send-email.js",
     "bootstrap-admins": "node ./scripts/bootstrap-admins.js",
-    "migrate-build-notification-settings": "node ./scripts/migrate-build-notification-settings.js"
+    "migrate-build-notification-settings": "node ./scripts/migrate-build-notification-settings.js",
+    "remove-bucket-website-configs": "node ./scripts/remove-bucket-website-configs.js"
```

Comment:
I'm not seeing this file.

Reply:
yikes, think i lost it when i renamed it, it's baaaaaaaaack